### PR TITLE
add riff

### PR DESCRIPTION
**Pull request by haiker2011**
* The riff CLI helps developers build and run functions using Knative.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -479,6 +479,10 @@ Please take a quick gander at the **[contribution guidelines](https://github.com
 - [microk8s](https://github.com/ubuntu/microk8s) - A kubernetes cluster in a snap [https://microk8s.io](https://microk8s.io/)
 - [minikube](https://github.com/kubernetes/minikube) - Run Kubernetes locally
 - [opencompose](https://github.com/redhat-developer/opencompose) - OpenCompose - A higher level abstraction for Kubernetes Resource
+- [operator-sdk](https://github.com/operator-framework/operator-sdk) - SDK for building Kubernetes applications. Provides high level APIs, useful abstractions, and project scaffolding. <https://coreos.com/operators>
```
Comment (rootsongjc) [inline README.md]: operator-sdk is duplicated.

File: README.md [hunk 2]
```diff
@@ -479,6 +479,10 @@ Please take a quick gander at the **[contribution guidelines](https://github.com
...
+- [prometheus-operator](https://github.com/coreos/prometheus-operator) - Prometheus Operator creates/configures/manages Prometheus clusters atop Kubernetes<https://coreos.com/operators/prometheus>
```
Comment (rootsongjc) [inline README.md]: prometheus-operator is duplicate too. The is a specific section for Kubernetes Operator.

File: README.md [hunk 3]
```diff
@@ -479,8 +479,6 @@ Please take a quick gander at the **[contribution guidelines](https://github.com
 - [microk8s](https://github.com/ubuntu/microk8s) - A kubernetes cluster in a snap [https://microk8s.io](https://microk8s.io/)
 - [minikube](https://github.com/kubernetes/minikube) - Run Kubernetes locally
 - [opencompose](https://github.com/redhat-developer/opencompose) - OpenCompose - A higher level abstraction for Kubernetes Resource
-- [operator-sdk](https://github.com/operator-framework/operator-sdk) - SDK for building Kubernetes applications. Provides high level APIs, useful abstractions, and project scaffolding. <https://coreos.com/operators>
-- [prometheus-operator](https://github.com/coreos/prometheus-operator) - Prometheus Operator creates/configures/manages Prometheus clusters atop Kubernetes<https://coreos.com/operators/prometheus>
 - [riff](https://github.com/projectriff/riff) - riff is for functions
```
Comment (rootsongjc) [inline README.md]: Is this the best category to place riff here?

Comment (haiker2011) [reply README.md]: May be in FaaS or Serverless category is better? form the description "The riff CLI helps developers build and run functions using Knative", I take riff as a tool to help build and run functions using Knative.

Comment (haiker2011) [reply README.md]: There is no Faas category, can you add this category? or FasS is included inServerless?

Comment (rootsongjc) [reply README.md]: Just put it in Serverless for now.

Comment (haiker2011) [reply README.md]: OK

**Pull request merged**